README.md

```diff
@@ -124,7 +78,7 @@ Make sure to set `nixpkgs.hostPlatform` in your `configuration.nix` to either `x
 
 ### Step 2. Installing `nix-darwin`
 
-Instead of using `darwin-installer`, you can just run `darwin-rebuild switch` to install nix-darwin. As `darwin-rebuild` won't be installed in your `PATH` yet, you can use the following command:
+Unlike NixOS, `nix-darwin` does not have an installer, you can just run `darwin-rebuild switch` to install nix-darwin. As `darwin-rebuild` won't be installed in your `PATH` yet, you can use the following command:
```

Comment:
The `nix-darwin` installer wasn’t anything like NixOS’s anyway. Perhaps we can just start “To install nix-darwin, you just run …`”.

Reply:
Fixed 👍